### DISCUSSION TITLE
The library is awesome

### DISCUSSION THREAD

**Discussion by ForNeVeR** [LanguageDev/Yoakke#119] [General]
That's it. That's what I wanted to say.

As a somewhat of a language enthusiast who had to spent a fair amount of time earlier developing parsers for (most of the time) primitive yet fully functional languages (using various approaches, from parser generation tools to parser combinator libraries), I am very impressed with Yoakke. It helps me immensely to roll out a full parser in a matter of hours, with very little effort.

Thanks a lot for publishing it.

**Comment by LPeter1997** [comment]
Thank you so much! I enjoy working on it so I'm happy others find it pleasant to use.

I'm hoping that one day I'll build up more complete modules to the level of the parser, so it could provide the joy in all aspects of compiler development.
